pymc/tests/distributions/test_timeseries.py

```diff
@@ -36,9 +37,50 @@
 )
 from pymc.model import Model
 from pymc.sampling import draw, sample, sample_posterior_predictive
-from pymc.tests.helpers import select_by_precision
-from pymc.tests.test_distributions_moments import assert_moment_is_expected
-from pymc.tests.test_distributions_random import BaseTestDistributionRandom
+from pymc.tests.distributions.util import (
+    BaseTestDistributionRandom,
+    Nat,
+    R,
+    Rplus,
+    Vector,
+    assert_moment_is_expected,
+    check_logp,
+)
+from pymc.tests.helpers import SeededTest, select_by_precision
+
+
+class TestMatchesScipy:
```

Comment:
This is repeated.  You can simply put the `test_logp` inside the TestGaussianRandomWalk class, not need for the MatchesScipy class